_posts/2025-03-20-phased-strimzi-upgrade-example.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: post
+title: "Phased upgrades of Strimzi managed Kafka fleets"
+date: 2025-03-20
+author: tom_cooper
+---
+
+## TL;DR
+
+When you upgrade an instance of the Strimzi Cluster Operator you will force **all** the Kafka clusters that it manages to roll. 
+For large fleets this could be quite disruptive.
+It would be nice to be able to decide which Kafka clusters upgrade to the new Strimzi version and when.
+
+This blog provides an example of how phased Strimzi upgrades can be performed.
+By using kustomize to create a deployment that allows multiple Strimzi versions to run concurrently and the `STRIMZI_CUSTOM_RESOURCE_SELECTOR` to allow us to specify which Strimzi instance manages which Kafka cluster.
+
+An [example repository](https://github.com/tomncooper/strimzi-phased-upgrade) is provided showing the final setup and providing example scripting and templates for adding new Strimzi versions.
+
+## Background
```

Comment:
If you move all headers one level down (i.e. `###` instead of `##`, `####` instead of `###`), the actual blog post on the web will look much better and the headers will fit with the rest of the website.